charts/umbrella/values.yaml

```diff
@@ -772,20 +795,20 @@ dataconsumerTwo:
   seedTestdata: false
   nameOverride: dataconsumer-2
   secrets:
-    edc-miw-keycloak-secret: changeme  # TODO switch to existing user
+    edc-miw-keycloak-secret: WUXpQx1aIclA7enqtk4o2uvLDLMreUMI
   tractusx-connector:
     nameOverride: dataconsumer-2-edc
     participant:
-      id: BPNL000000000000
+      id: BPNL00000003B6LU
```

Comment:
could you please take BPNL00000003AVTH, same as in adopter-dataexchange values file